shift_test.go

```diff
@@ -291,39 +290,43 @@ func TestGenFSM_Update(t *testing.T) {
 
 	var unknownShiftStatus TestStatus = 999
 	tests := []struct {
-		name        string
-		from        shift.Status
-		to          shift.Status
-		expectedErr error
+		name   string
+		from   shift.Status
+		to     shift.Status
+		expErr error
+		expKVs j.MKV
```

Comment:
`expKVs` is not used in any assertion, should there be a check to verify the values?

Reply:
Ah, good spot Pierre! I want to write a pre-commit hook to check for unused test fields like this, not sure how easy it will be though!